### PR TITLE
various fixes

### PR DESCRIPTION
**Pull request by romainmenke**
- `double-position-gradients` transforms broke with CSS functions.
- `package.json` error in `gradients-interpolation-method`
- small tweaks and `color-mix` in `progressive-custom-properties`

### CODE REVIEW

File: plugins/postcss-double-position-gradients/test/basic.expect.css [hunk 1]
```diff
@@ -91,3 +91,16 @@
...
+		from 3deg,
+		hsl(200, 100%, 50%) 0deg,hsl(200, 100%, 50%) 15deg,
+		hsl(200, 100%, 60%) 10deg, hsl(200, 100%, 60%) 30deg
+	);
+	background: repeating-conic-gradient(
+		from 3deg,
+		hsl(200, 100%, 50%) 0deg 15deg,
+		hsl(200, 100%, 60%) 10deg 30deg
+	);
+}
```
Comment (romainmenke) [inline plugins/postcss-double-position-gradients/test/basic.expect.css:106]: The value in `postcss-value-parser` for `hsl(200, 100%, 60%)` is `hsl`.
So any double position gradients with CSS functions were broken :/

```css
.repeating-conic {
	background: repeating-conic-gradient(
		from 3deg,
		hsl(200, 100%, 50%) 0deg,hsl 15deg,
		hsl(200, 100%, 60%) 10deg, hsl 30deg
	);
}
```

File: plugins/postcss-progressive-custom-properties/src/match.ts [hunk 2]
```diff
@@ -12,12 +12,23 @@ export function matches(a, b) {
...
+			let ia = i;
+			let ib = i;
+
+			while (a.nodes[ia] && a.nodes[ia].type === 'space') {
+				ia++;
+			}
+
+			while (b.nodes[ib] && b.nodes[ib].type === 'space') {
+				ib++;
+			}
```
Comment (romainmenke) [inline plugins/postcss-progressive-custom-properties/src/match.ts:25]: `space` nodes are now ignored when matching.
Makes it easier to match cases with irregular spacings : `rgba(255,0, 255 , 1)`

File: plugins/postcss-gradients-interpolation-method/package.json [hunk 3]
```diff
@@ -23,13 +23,13 @@
 		"README.md",
 		"dist"
 	],
-	"peerDependencies": {
-		"postcss": "^8.3"
-	},
-	"devDependencies": {
+	"dependencies": {
```
Comment (romainmenke) [inline plugins/postcss-gradients-interpolation-method/package.json:26]: Temporary laps in focus on my end?

**Review by Antonio-Laguna — APPROVED** [review]

**Comment by Antonio-Laguna** [comment]
LGTM! Thanks!

**Pull request merged**